packages/docmanager-extension/schema/plugin.json

```
@@ -149,6 +149,13 @@
       "title": "Rename Untitled File On First Save",
       "description": "Whether to prompt to rename untitled file on first manual save.",
       "default": true
+    },
+    "maximalRecents": {
```

Another suggestion for the name of this setting:

```suggestion
    "maxNumberRecents": {
```